tyrian/js/src/main/scala/tyrian/runtime/CmdHelper.scala

```diff
@@ -1,15 +1,17 @@
 package tyrian.runtime
 
 import cats.Applicative
+import cats.effect.std.Dispatcher
 import tyrian.Cmd
 
 import scala.annotation.tailrec
 
 object CmdHelper:
 
-  def cmdToTaskList[F[_]: Applicative, Msg](cmd: Cmd[F, Msg]): List[F[Option[Msg]]] =
+  def cmdToTaskList[F[_]: Applicative, Msg](cmd: Cmd[F, Msg],
+    disp: Dispatcher[F]): List[F[Option[Msg]]] =
```

Comment:
Our PRs are now conflicting! 😁 
- https://github.com/PurpleKingdomGames/tyrian/pull/171

Reply:
Ooooh, interesting! I'll take a look. I'm curious to learn what you changed.